GSoC25/TestFiles/pipeline.py

```
@@ -0,0 +1,454 @@
+    PROJECT_ROOT = Path().resolve()
+if str(PROJECT_ROOT) not in sys.path:
+    sys.path.append(str(PROJECT_ROOT))
+
+# Import NEF components
+from GSoC24.Data.collector import get_text_of_wiki_page
+
+
+class RedisEntityLinking:
+    """Redis-based entity linking with fallback to LLM"""
+    
+    def __init__(self, host='91.99.92.217', port=6379, password='NEF!gsoc2025'):
```

Ruff (0.14.2)

Possible hardcoded password assigned to function default: "password"

(S107)